src/test/java/TestPrinterFormatter.java

```diff
@@ -0,0 +1,36 @@
+import org.junit.Test;
+import training.supportbank.PrinterFormatter;
+
+import java.math.BigDecimal;
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class TestPrinterFormatter {
+    private static Map<String, BigDecimal> fakeMapInput = new LinkedHashMap<>();
+    private static BigDecimal fakeBigDecimal;
+
+    @Test
+    public void formatterConvertToCurrencyOutputsStringCurrencyValue() {
+        fakeBigDecimal = new BigDecimal("5.00");
+        PrinterFormatter formatter = new PrinterFormatter();
+        String currencyValue = formatter.convertToCurrency(fakeBigDecimal, Locale.UK);
+        assertThat(currencyValue).isEqualTo("£5.00");
+    }
+
+    @Test
+    public void formatterCreateScriptCreatesCreditString() {
+        fakeBigDecimal = new BigDecimal("5.00");
+        fakeMapInput.put("John D", fakeBigDecimal);
+        fakeBigDecimal = new BigDecimal("-3.00");
```

Comment:
I think it's a bit confusing to reuse this variable, because it looks like you're updating the state of something but really it's just being used for convenience.

I'd be tempted to just inline it like this:

```
fakeMapInput.put("John D", BigDecimal.valueOf(5));
```

Reply:
used `new BigDecimal("5.00")` syntax to keep with current structure of using strings. This will change in future to allow use of valueOf(Integer) syntax.